INTYGFV-14993: Layout issues for UeMedicalInvestigation

SamuelTilly opened the pull request.

ChrilleSigerhed reviewed (commented):
Looks great with the changes made to the layout. Are we able to help the user more with the "Ange utredning eller underlag" section? Maybe add ... and a tooltip on mouseover?

![image](https://user-images.githubusercontent.com/57347249/212295791-725a8878-3218-4dbe-888d-1627e1b7f10b.png)

SamuelTilly commented:
> Looks great with the changes made to the layout. Are we able to help the user more with the "Ange utredning eller underlag" section? Maybe add ... and a tooltip on mouseover?
> 
> ![image](https://user-images.githubusercontent.com/57347249/212295791-725a8878-3218-4dbe-888d-1627e1b7f10b.png)

Good call!, I'll see what I can do

SamuelTilly commented:
![image](https://user-images.githubusercontent.com/28404/212310936-e831964a-0b2f-4984-a92e-f262937dcb89.png)

Added ellipsis, tooltip and fixed dropdown padding.

ChrilleSigerhed reviewed (approved):
Nice! Approved!
